src/client/utils/dom.js

```diff
@@ -761,3 +775,13 @@ export function getFileInputs (el) {
 export function getIframes (el) {
     return isIframeElement(el) ? [el] : getNativeQuerySelectorAll(el).call(el, 'iframe,frame');
 }
+
+export function hasElementProperty (el, prop) {
+    if (el[prop] === void 0)
+        return false;
+
+    if (isSVGElement(el))
+        return DEFAULT_SVG_ELEMENT_TEXT_ATTR_VALUES[prop] !== void 0;
+
+    return DEFAULT_HTML_ELEMENT_TEXT_ATTR_VALUES[prop] !== void 0;
+}
```

Comment:
I think that the `elementHasTextProperty` is an utility function and it must be stored in the `utils`.